custom/icds_reports/views.py

```diff
@@ -2379,3 +2380,51 @@ def get(self, request, *args, **kwargs):
             }
         }
         return JsonResponse(data=response_json)
+
+
+@location_safe
+@method_decorator([api_auth, toggles.ICDS_DEMOGRAPHICS_API.required_decorator()], name='dispatch')
+class DemographicsAPI(View):
```

Comment:
Might be nice to call this BiharDemographicsAPI, to be clear here (and because that phrase is used elsewhere)

Reply:
We have a bunch of very similar API views (and data fethcing functions), is it possible to share any of this functionality between them. It feels like duplicate a bunch every time

Reply:
yep. created one base class and used for this view and for Disha API for now